cost/aws/schedule_instance/README.md

```diff
@@ -67,14 +69,18 @@ This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Auto
   - `ec2:DescribeInstances`
   - `ec2:StartInstances`
   - `ec2:StopInstances`
-  - `ec2:TerminateInstances`
-  - `ec2:CreateTags`
   - `ec2:DeleteTags`
   - `ec2:DescribeRegions`
-  - `kms:CreateGrant`*
-  - `kms:Decrypt`*
+  - `kms:CreateGrant`**
```

Comment:
```suggestion
  - `kms:CreateGrant`‡
```
We may need to update the policy permissions script to support the new symbol (whether double-asterisk or cross-dagger), just so the note is added to the policy permissions list json/yaml

Can add this in a separate PR